COUETTE/COUETTE-IOC-02App/Db/Makefile

```diff
@@ -0,0 +1,18 @@
+TOP=../..
```

Comment:
I'm not sure this second IOC makes sense as it is an LvDCOM and only has one underlying vi?